packages/web-frontend/src/components/MapOverlayHierarchy/MapOverlayHierarchy.js

```diff
@@ -0,0 +1,61 @@
+/**
+ * Tupaia
+ * Copyright (c) 2017 - 2022 Beyond Essential Systems Pty Ltd
+ */
+
+import React from 'react';
+import { HierarchyItem } from './HierarchyItem';
+
+export const MapOverlayHierarchy = ({
```

Comment:
This is pulling out lines [149-204](https://github.com/beyondessential/tupaia/pull/3774/files#diff-ec94902c8f0c53a834ef9b6e3a00222079b45acc2e735ea8b165e6c1fd4cb64dL149-L204) from MapOverlayBar.js so they can be shared by the mobile version